grype/db/v6/enumerations.go

```diff
@@ -57,6 +60,11 @@ const (
 	NotAffectedFixStatus FixStatus = "not-affected"
 )
 
+const (
+	// AdvisoryReferenceTag is a reference to a vulnerability advisory
+	AdvisoryReferenceTag string = "advisory"
```

Comment:
Should there be more of these?

Reply:
A reference tag is a string we expect to be used often to tag a reference in https://github.com/anchore/grype/blob/main/grype/db/v6/blobs.go#L51.

What's an Advisory exactly? For example, a Red Hat Security Advisory (RHSA) is a list of patches, but a GitHub Security Advisory is a vulnerability description. Is `advisory` the right tag name? What kind of links or going to be tagged with this tag name?

Reply:
I think I'll add these in another PR, but agree with the overall question